assets/chezmoi.io/docs/reference/templates/functions/setValueAtPath.md

```diff
@@ -0,0 +1,14 @@
+# `setValueAtPath` *path* *value* *dict*
+
+`setValueAtPath` modifies *dict* to set the value at *path* to *value* and
+returns *dict*. *path* can be either a string containing a `.`-separated list of
+keys or a list of keys. It is an alternative to [sprig's `set`
+function](http://masterminds.github.io/sprig/dicts.html) with a different
+argument order that supports pipelining.
+
+!!! example
+
+    ```
+    {{ dict | setValueAtPath "key1" "value1" | setValueAtPath "key2.nestedKey" "value2" | toJson }}
+    {{ dict | setValueAtPath (list "key2" "nestedKey") "value2" | toJson }}
+    ```
```

Comment:
Should we show the sample output?

More to the point, would `cat '{}' | chezmoi execute-template "{{ .chezmoi.stdin | fromJson | setValueAtPath "key1" "value" }}` work, as that would make it easier to demonstrate.

Reply:
All the other template function examples don't include the output.